lib/galaxy/model/__init__.py

```diff
@@ -5514,6 +5514,7 @@ def input_associations(self):
 
     def to_dict(self, view='collection', value_mapper=None, step_details=False, legacy_job_state=False):
         rval = super().to_dict(view=view, value_mapper=value_mapper)
+        rval['stored_workflow_id'] = self.workflow.stored_workflow.id
```

Comment:
@simonbray Could we drop this then ? I am worried about clients consuming this, because I feel like that should be deprecated ASAP.

Reply:
We can remove it - of course using `workflow_id` is preferable.

But `stored_workflow_id` is the default for everything as far as I can see: https://github.com/galaxyproject/galaxy/blob/c799e4a455562ba3ccc7058a6c7ec6ec6755eaef/lib/galaxy/webapps/galaxy/api/workflows.py#L1451

which is why I felt it was useful to provide it.

Reply:
It's the default because otherwise every existing external call would break. In most cases I would prefer not to use `instance`, but to use `workflow_id` and `stored_workflow_id` as the public parameters. I have not checked in how far we already use `workflow_id` in cases where this should be the `stored_workflow_id`. If we most use `id` we can make that change, otherwise we're stuck with the instance parameter.